tasks/gulp/copy-assets.js

```diff
@@ -23,6 +23,7 @@ gulp.task('copy-assets:local', (done) => {
   mergedStream.on('queueDrain', (done))
 })
 
+// TODO: do we need any of these?
```

Comment:
I hope not

Reply:
Will remove these against a seperate ticket